srcpkgs/wxWidgets-gtk3/template

```diff
@@ -0,0 +1,71 @@
+# Template file for 'wxWidgets-gtk3'
+pkgname=wxWidgets-gtk3
+version=3.0.4
+revision=1
+configure_args="--with-gtk=3 --enable-unicode --with-opengl --with-sdl --with-libmspack
+ --with-libnotify --enable-mediactrl"
+build_style=gnu-configure
+wrksrc="wxWidgets-${version}"
+hostmakedepends="pkg-config"
+makedepends="SDL2-devel gst-plugins-base1-devel gstreamer1-devel gtk+3-devel
+ libmspack-devel libnotify-devel tiff-devel webkit2gtk-devel"
+short_desc="The wxWidgets GUI toolkit library compiled with gtk3 support"
+maintainer="Helmut Pozimski <helmut@pozimski.eu>"
+license="wxWindows"
+homepage="https://www.wxwidgets.org"
+distfiles="https://github.com/wxWidgets/wxWidgets/releases/download/v${version}/wxWidgets-${version}.tar.bz2"
+checksum=96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0
+
+wxWidgets-common_package() {
+	short_desc+=" - common files"
+	pkg_install() {
+		vmove "usr/lib/libwx_baseu*.so.*"
+	vmove usr/share
+	}
+}
+
+wxWidgets-common-devel_package() {
+	depends="wxWidgets-common>=${version}_${revision}"
+	short_desc+=" - common development files"
+	pkg_install() {
+		vmove usr/bin/wxrc-3.0
+		ln -sf wxrc-3.0 ${PKGDESTDIR}/usr/bin/wxrc
```

Comment:
The above two lines should probably be just
```
vmove "usr/bin/wxrc*"
```
because it seems the symlink already exists and is left in wxWidgets-gtk3-devel.

Reply:
I'm have just been working on that, didn't notice the leftover symlink earlier. My first thought was to add another vmove line but that's of course better. Will push the amended template shortly.